source/ServerInput.cpp

```diff
@@ -1,12 +1,12 @@
-#include "ServerInput.hpp"
+#include "InputManager.hpp"
 #include "Server.hpp"
 
-ServerInput::ServerInput() {
-  SET_LISTENER(ServerInput, ServerInputListeners, new_input, server_new_input);
+InputManager::InputManager() {
```

Comment:
```suggestion
InputManager::InputManager()
{
```
since we're doing cleanup why not